pyerrors/input/openQCD.py

```diff
@@ -618,16 +730,14 @@ def read_qtop(path, prefix, c, dtr_cnfg=1, version="openQCD", **kwargs):
     r_stop_index = []
     deltas = []
     configlist = []
+    if not zeuthen:
+        obspos += 8
```

Comment:
Is this correct for all file and code versions? I think it might only be true for the `sfqcd` files. At least this is how I had implemented it in lines 627-630 of the old code version.

Reply:
I think the variable `obspos` is only relevant for `sfqcd` files. For the `openQCD` files the function currently always extracts the topological charge based on the gradient flow regardless of what is specified for `obspos` or `Zeuthen_flow`.

Reply:
Yes, you are right, I did not see that.